editor/src/clj/editor/atlas.clj

```diff
@@ -530,28 +594,18 @@
   (output texture-profile g/Any (g/fnk [texture-profiles resource]
                                   (tex-gen/match-texture-profile texture-profiles (resource/proj-path resource))))
 
-  (output all-atlas-images           [Image]             :cached (g/fnk [animations]
-                                                                   (into [] (comp (mapcat :images) (distinct)) animations)))
-
-  (output texture-set-data-generator g/Any (g/fnk [_node-id animations all-atlas-images extrude-borders inner-padding margin resource :as args]
-                                                  (or (when-let [errors (->> [[margin "Margin"]
-                                                                              [inner-padding "Inner Padding"]
-                                                                              [extrude-borders "Extrude Borders"]]
-                                                                             (keep (fn [[v name]]
-                                                                                     (validation/prop-error :fatal _node-id :layout-result validation/prop-negative? v name)))
-                                                                             not-empty)]
-                                                        (g/error-aggregate errors))
-                                                      {:f    generate-texture-set-data
-                                                       :args (assoc args
-                                                               :workspace (resource/workspace resource))})))
-
-  (output texture-set-data g/Any               :cached (g/fnk [texture-set-data-generator] (call-generator texture-set-data-generator)))
-  (output layout-size      g/Any               (g/fnk [texture-set-data] (:size texture-set-data)))
```

Comment:
The `texture-set-data-generator` has now become the `layout-data-generator` below. The `texture-set-data` and `layout-size` implementations have changed and are a few lines down from here.